### DISCUSSION TITLE
R1 R2 and so on in SMILES.

### DISCUSSION THREAD

**Discussion by khoroshyy** [rdkit/rdkit#4951] [Q&A] [answered]
Hi.
I cannot figure out if smiles support radicals to make something like this:

CH3-R1
R1-CH2-R2.
Thanks.

**Comment by dehaenw** [comment, marked as answer]
Hi @khoroshyy ,
in smiles you can use dummy atoms for this purpose, you can use `*` instead of an element symbol, so for example `c1ccccc1*` for a benzene ring with a dummy attached. See the informative blog post here which makes use of labelled dummies for a more advanced case https://greglandrum.github.io/rdkit-blog/tutorial/prototypes/drawing/2021/08/07/rgd-and-highlighting.html

**Reply by khoroshyy** [reply]
Thanks a lot.
I remember seeing it somewhere but was not able to find it again.
